uwblueprint/ccbc
# Tahmeed/client demo

## Notion ticket link

[Invite New Admin Flow](https://www.notion.so/uwblueprintexecs/Task-Board-db95cd7b93f245f78ee85e3a8a6a316d)



## Implementation description
* Added a Modal to invite new admin from the navbar
* Created a new InviteAdminModal component in the common folder so we can use this component elsewhere if needed
* Created a new InviteAdminModalStyle class which will have the styles for this modal. TODO: Can't style this custom modal. I can only seem to style all Modals in general, but when I create a custom component for the modal, the styles don't seem to be working



## Steps to test
1. Log in to an account to see the home page + navbar
2. Go to profile icon and click invite new admin
3. enter the details for the new account
4. log out of your current account
5. open the email you entered and click on the verify link
6. Go through the steps -> you should now be logged in with your new account



## What should reviewers focus on?
* Are the new files created consistent with the file/folder structure we want to maintain on the front end
* How do style the custom component so we can apply specific styles to this component only?


## Checklist
